vsts/vsts.yaml

```diff
@@ -1,17 +1,34 @@
 name: $(BuildID)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
 
+# https://docs.microsoft.com/azure/devops/pipelines/process/runtime-parameters
+parameters:
+- name: jobTimeoutInMinutes
+  displayName: Timeout for each job
+  type: number
+  default: 75
+- name: tests
+  displayName: Run all tests, include clearing (if unchecked, clear service resources only).
+  type: boolean
+  default: true
+  
+variables:
+  ${{ if parameters.tests }}:
+    testConfig: "-run"
+  ${{ else }}:
+    testConfig: "-clear"
+
 resources:
   - repo: self
     clean: true
 
 jobs:
   ### Linux build ###
   - job: LINUX
-    displayName: Linux
+    displayName: Linux (timeout=${{ parameters.jobTimeoutInMinutes }}mins)
```

Comment:
Seems odd to make that one property part of the title.

Reply:
If we need this printed somewhere, I'd just document it in a step.

Reply:
I thought it allows us to see if the parameter is set, seeing the timeout knowing if custom parameter is taken on the job.
Alternative is to print this out on the log. We can change it if this info is not helpful.